tool/utils.ts

```diff
@@ -262,6 +262,7 @@ function buildEmbeddedProtocol(repoPath: string): void {
       --plugin="protoc-gen-ts=${pluginPath}" \
       --js_out="import_style=commonjs,binary:." \
       --ts_out="." \
```

Comment:
The `js_out` and `ts_out` should probably match `proto_path`.

Reply:
I think I actually want to make them explicitly `build/embedded-protocol`. Currently this is relying on the input path having that structure, which isn't a guarantee I want to enforce.